spec/features/item_registration_spec.rb

```diff
@@ -78,7 +79,9 @@
       'project' => 'special division : project #4',
       'rights' => 'default',
       'source_id' => 'source:id1',
-      'tag' => ['Process : Content Type : Book (ltr)', 'tag : test', "Registered By : #{user.sunetid}"],
+      'content_type' => 'https://cocina.sul.stanford.edu/models/book',
+      'viewing_direction' => 'left-to-right',
+      'tag' => ['tag : test', "Registered By : #{user.sunetid}"],
       'workflow_id' => 'goobiWF'
```

Comment:
verify the params come through ok (but without the Process : Content Type tag, which we don't need or want anymore)